apps/app-e2e/hardhat.config.ts

```diff
@@ -0,0 +1,85 @@
+import '@typechain/hardhat'
```

Comment:
Jonathan just provide a runnable code this afternoon, I'm researching it and will cleanup them after that.

Reply:
This needs to be a hardhat project since the tests rely on hardhat components and will fail if this is removed.  @sampullman let me know if you know of a workaround for this or if you are referring to something else.  Thanks!